features/scripts/clear_local_maven_repo.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-mv ~/.m2/repository/com/bugsnag/bugsnag-android-gradle-plugin ~/.Trash/
+mv ~/.m2/repository/com/bugsnag/bugsnag-android-gradle-plugin ~/.Trash/ || true
```

Comment:
Why not just `rm -rf` here? That would avoid having to worry about whether the file exists.

Reply:
I thought it would be better having this action be reversible in case something went horrendously wrong with the script/maze-runner and it deleted files it wasn't supposed to. If you're happy I can change it to `rm -rf`